README.md

```diff
@@ -1,5 +1,9 @@
 # Hyperledger Blockchain Integration Framework <!-- omit in toc -->
 
+[![Build status]](https://travis-ci.org/denis-yu-glotov/blockchain-integration-framework)
+
+[Build status]: https://travis-ci.org/denis-yu-glotov/blockchain-integration-framework.svg?branch=denis/travis-ci
```

Comment:
Is this correct for this repo? Looks like it might have to do with the changes on your fork and branch.

Reply:
Thanks for reminder, this should have been updated last - because Travis is not yet enabled for `hyperledger-labs/blockchain-integration-framework` repo. Just fixed it it my latest commit.
BTW, do you know who can enable Travis for our repo?
And I will meanwhile update my commits with signatures.

Reply:
I flipped the switch. Let me know if there is something else that is needed beyond that.

Reply:
All done now. May I merge?

Reply:
@denis-yu-glotov  has addressed feedback